app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1093,43 +1093,6 @@ class PatientRepositoryAndroidTest {
     verifyRecentPatientOrder(patientWithBp2, facilityUuid = facility2Uuid)
```

Comment:
Please amend the first commit itself to fix this broken test.

Reply:
done